test/units/srcset_test.rb

```diff
@@ -68,7 +68,34 @@ def test_srcset_signs_urls
             }
         end
 
+        def test_srcset_has_variable_qualities
+            i = 0
+            srcset.split(',').map { |src|
+                assert_includes src, "q=#{DPR_QUALITY[i]}"
+                i += 1
+            }
+        end
+
+        def test_srcset_respects_overriding_quality
+            quality_override = 100
+            srcset = Imgix::Client.new(host: 'testing.imgix.net', secure_url_token: 'MYT0KEN', include_library_param: false).path('image.jpg').to_srcset(w:100, q:quality_override)
+
+            srcset.split(',').map { |src|
+                assert_includes src, "q=#{quality_override}"
+            }
+        end
+
+        def test_disable_variable_qualities
+            srcset = Imgix::Client.new(host: 'testing.imgix.net', secure_url_token: 'MYT0KEN', include_library_param: false).path('image.jpg').to_srcset(w:100, options: { disable_variable_qualities: true })
+
+            srcset.split(',').map { |src|
+                assert(not(src.include? "q="))
+            }
+        end
+
```

Comment:
Can we add one more test to make sure an overriding `q` param is still respected even if `disable_variable_qualities` is false?